addons/hr_expense/tests/test_expenses.py

```diff
@@ -1098,6 +1098,10 @@ def test_corner_case_expense_reported_cannot_be_zero(self):
         # CASE 8: ALLOWS Setting the amounts to 0 while unlinking the expense sheet
         expense.write({'total_amount_currency': 0.0, 'total_amount': 0.0, 'sheet_id': False})
 
+        # CASE 9: FORBIDS Creating an expense with amounts to 0 from the list view in the expense sheet
+        with self.assertRaises(UserError):
+            self.create_expense(values={'total_amount': 0.0, 'total_amount_currency': 0.0}, ctx={'check_total_amount_not_zero': True})
```

Comment:
I'd prefer a test using a Form, to be sure we are actually testing the user flow :)